### ISSUE TITLE
Fails on WildFly 11.0.0

### ISSUE THREAD

**Issue by t1** [wildfly-extras/wildfly-microprofile-config#22]
When I install the extension on a WildFly 11.0.0, I get this exception:

```
java.lang.RuntimeException: WFLYCTL0435: Attribute class of resource [
    ("subsystem" => "microprofile-config"),
    ("config-source" => "*")
] was not marked as reload required, either set the RESTART_ALL_SERVICES flag, or register a custom write handler.
```

I don't understand, what this all means, so I can't help much to resolve the issue.

**Comment by jmesnil** [comment]
This should be fixed by #23 (WildFly 11 has further check on the management model of its subsystems).

**Comment by t1** [comment]
Okay... thanks
